tmail-backend/tmail-third-party/openpaas/src/test/java/com/linagora/tmail/OpenPaasRabbitMQExtension.java

```diff
@@ -0,0 +1,43 @@
+package com.linagora.tmail;
+
+import org.apache.james.GuiceModuleTestExtension;
+import org.apache.james.backends.rabbitmq.DockerRabbitMQ;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ParameterContext;
+import org.junit.jupiter.api.extension.ParameterResolutionException;
+
+import com.google.inject.Module;
+
+public class OpenPaasRabbitMQExtension implements GuiceModuleTestExtension {
+    private final DockerRabbitMQ dockerRabbitMQ = DockerRabbitMQ.withoutCookie();
```

Comment:
IMO this extension is not needed we could have 2 connections to the main RabbitMQ extension and we would still be able to assert if we want that the mail RabbitMQChannelPool is distinct from the one annotated with `@Named("openpaas")`